agent/functional_tests/tests/functionaltests_unix_test.go

```diff
@@ -618,3 +618,197 @@ func TestNetworkModeBridge(t *testing.T) {
 	err := networkModeTest(t, agent, "bridge")
 	require.NoError(t, err, "Networking mode bridge testing failed")
 }
+
+// TestCustomAttributesWithValidOptions tests the ECS_INSTANCE_ATTRIBUTES
+// upon agent registration with a key, value pair
+func TestCustomAttributesWithValidOptions(t *testing.T) {
+	len := 25
+	key, value := GetRandomString(len), GetRandomString(len)
+
+	customAttribute := fmt.Sprintf("{\"%s\": \"%s\"}", key, value)
```

Comment:
When we have those requirements, we can write this code if needed, I don't think we need it now as it's not really giving us any value.